servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/OutlierDetectorConfig.java

```diff
@@ -512,6 +526,21 @@ public Builder cancellationIsError(final boolean cancellationIsError) {
             return this;
         }
 
+        /**
+         * Set the penalty factory to apply to outstanding requests.
+         * The EWMA penalty to apply to endpoints when there are outstanding requests. By penalizing endpoints with
+         * outstanding load the traffic distribution will be smoother for algorithms that consider load metrics.
+         * Higher penalties will favor a more even request distribution while lower penalties will bias traffic toward
+         * endpoints with better performance.
+         * Defaults to {@value DEFAULT_PENDING_REQUEST_PENALTY}.
+         * @param pendingRequestPenalty the penalty factory to apply for outstanding load.
+         * @return {@code this}
+         */
+        public Builder pendingRequestPenalty(final long pendingRequestPenalty) {
```

Comment:
Based on our offline discussion: consider renaming this option to `concurrentRequestPenalty` (to align with h2 max_concurrent_streams) or `outstandingRequestPenalty`. 
Term "pending" makes an impression that these requests are pending in a queue for execution, but we don't have such a queue in ST at all.